testapps/test_pillowtop/tests/test_form_pillow.py

```diff
@@ -22,15 +30,23 @@ def setUp(self):
         super(FormPillowTest, self).setUp()
         FormProcessorTestUtils.delete_all_xforms()
         self.pillow = get_xform_pillow(skip_ucr=True)
-
         factory = AppFactory(domain=self.domain)
         self.app = factory.app
         self.app.save()
 
+    @classmethod
+    def setUpClass(cls):
```

Comment:
setUpClass must call super
```suggestion
    def setUpClass(cls):
        super().setUpClass()
```